### PR TITLE
fix(ui): set ui specific i18n storage key

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

set ui specific i18n storage key
to distinguish it from ac settings. 

image

## Testing

@logto-io/eng

### CODE REVIEW

File: packages/ui/src/i18n/init.ts [hunk 1]
```diff
@@ -11,6 +11,10 @@ const initI18n = async (languageSettings?: LanguageInfo) => {
     interpolation: {
       escapeValue: false,
     },
+    detection: {
+      lookupLocalStorage: 'i18nextUILng',
+      lookupSessionStorage: 'i18nextUILng',
```
Comment (gao-sun) [inline packages/ui/src/i18n/init.ts:16]:
```suggestion
      lookupLocalStorage: 'i18nextLogtoLng',
      lookupSessionStorage: 'i18nextLogtoLng',
```

Comment (simeng-li) [reply packages/ui/src/i18n/init.ts]: updated

File: packages/ui/src/i18n/init.ts [hunk 2]
```diff
@@ -11,6 +11,10 @@ const initI18n = async (languageSettings?: LanguageInfo) => {
     interpolation: {
       escapeValue: false,
     },
+    detection: {
+      lookupLocalStorage: 'i18nextLogtoUILng',
```
Comment (darcyYe) [inline packages/ui/src/i18n/init.ts:15]: should this be `i18nextLogtoUiLng`?

**Review by wangsijie — APPROVED** [review]

File: packages/ui/src/i18n/init.ts (hunk 2 above)
Comment (simeng-li) [reply packages/ui/src/i18n/init.ts]: updated

**Pull request merged**